README.md

```diff
@@ -1,9 +1,243 @@
 # ros_utilities
 Wrappers and other utilities for ROS2
 
+## Getting started
+
+### Process-wide APIs
+
+These APIs wrap [`rclpy`](https://github.com/ros2/rclpy) APIs to provide a simpler UX in common use cases.
+All APIs are thread-safe and exception-safe. Resource management is automatic. By default, callbacks are
+dispatched in the background, enabling both synchronous and asynchronous programming.
+
+These APIs are also quite handy to reconcile past [`rospy`](http://wiki.ros.org/rospy) experience with ROS 2.
+
+#### Setting up single node processes
+
+Just decorate your executable entrypoint (or `main` function) with `bdai_ros2_wrappers.process.main`:
```

Comment:
```suggestion
To create simple single node processes that can block in common sense ways similar to ROS1's `rospy`, just decorate your executable entrypoint (or `main` function) with `bdai_ros2_wrappers.process.main`:
```

Reply:
Can you also specify what the person should see if they run this program?  I actually don't know what `create_task` does...